docs/architecture/data-model.md

```diff
@@ -0,0 +1,105 @@
+---
+layout: default
+title: Data Model
+parent: Architecture
+description: lakeFS Data Model explained
+nav_order: 2
+has_children: false
+---
+# Data Model
+{: .no_toc }
+
+
+## Table of contents
+{: .no_toc .text-delta }
+
+1. TOC
+{:toc}
```

Comment:
This seems weird.  Is this something Jekyllish?

Reply:
This renders the top-of-page table of contents

Reply:
so, yes.